modules/installation-gcp-shared-vpc-permissions.adoc

```diff
@@ -0,0 +1,20 @@
+// This file is referenced in the following assembly:
+// installing/installing_gcp/installing-gcp-shared-vpc.adoc
+
+:_content-type: PROCEDURE
+[id="installation-gcp-shared-vpc-permissions_{context}"]
+= Required GCP permissions for shared VPC installation
+
+When you attach the `Owner` role to the service account that you create, you
+grant that service account all permissions, including those that are required to
+install {product-title}. To deploy an {product-title} cluster into a shared VPC, the
+service account requires the following permissions in the host project.
+
+.Required roles for creating network resources during installation
+* Compute Network User
+* Compute Security Admin
+[NOTE]
```

Comment:
```suggestion
+
[NOTE]
```
Added LGTM and then noticed that the formatting is a little off on this. Add a `+` before the note